pySDC/projects/PinTSimE/estimation_check.py

```diff
@@ -297,17 +324,15 @@ def differences_around_switch(
         times_adapt = [me[0] for me in get_sorted(stats_adapt, type='u', recomputed=False)]
         times_SE_adapt = [me[0] for me in get_sorted(stats_SE_adapt, type='u', recomputed=False)]
 
-        diffs_true_at.append(
-            [diff_SE[m] for m in range(len(times_SE)) if np.isclose(times_SE[m], t_switch, atol=1e-15)][0]
-        )
+        diffs_true_at.append([diff_SE[m] for m in range(len(times_SE)) if abs(times_SE[m] - t_switch) <= 1e-17][0])
```

Comment:
testing if the difference between two floats is less than machine precision seems fishy too me.

Reply:
I agree, it's fishy.. it also works with a larger tolerance.